components/public-api/typescript/BUILD.yaml

```diff
@@ -4,13 +4,26 @@ packages:
     deps:
       - components/gitpod-protocol:lib
     srcs:
-      - "src/*.ts"
-      - "src/*.js"
+      - src/**
       - package.json
+      - tsconfig.json
     config:
       packaging: library
       dontTest: true
       commands:
         build: ["yarn", "build"]
       yarnLock: ${coreYarnLockBase}/../yarn.lock
       tsconfig: tsconfig.json
+
+  - name: publish
```

Comment:
How is this triggered? We already have this:

[mp/public-api-publish-npm](https://github.com/gitpod-io/gitpod/blob/c887240be815c345efbba3fcc2660935637bd767/components/BUILD.yaml#L81-L89)

Does that need to be updated

Reply:
Not sure, mirrors supervisor-api setup